app/src/main/AndroidManifest.xml

```diff
@@ -161,9 +161,7 @@
 
         <activity
             android:name=".ui.account.AccountSelectActivity"
-            android:configChanges="orientation|screenSize|smallestScreenSize|screenLayout|keyboard|keyboardHidden|navigation"
```

Comment:
any reason why you would want to delete all of these? It will recreate the entire activity when you rotate the screen, so if you want to input your pin and rotate the screen it will disappear for example. I see no reason why you would want to make it act like this

Reply:
It was unintentional to remove the whole line. I meant to just remove a couple meaningless ones but selected the line apparently. My bad.